evoting/lib/election.go

```diff
@@ -10,6 +10,15 @@ import (
 	"github.com/dedis/cothority/skipchain"
 )
 
+const (
```

Comment:
These constants should be typed, so that they can only be used where they are expected to be. See https://golang.org/doc/effective_go.html#constants.

Thus:
```
type ElectionState uint32
const (
  Running ElectionState = iota + 1
)
```

Sometimes at the edge of the system (i.e. where protobufs are sent over websockets) this can't work and you want to use integers instead, but I don't think this is that case.